linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/ignition/shared.yml

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          source: data:,options%20%7B%20include%20/etc/crypto-policies/back-ends/bind.config%3B%20%7D%0A
```

Comment:
Since this follows the ``ini`` format, the include needs be in the ``[ crypto_policy ]`` section.  Something like:

`````
[ crypto_policy ]
.include /etc/crypto-policies/back-ends/openssl.config
`````

Reply:
Ah okay I will change that.